src/_includes/soft_mod_cheat_sheet_cta_banner.html

```diff
@@ -1,6 +1,6 @@
 <div class="container section g-pt-200 g-pb-100">
   <div class="soft-mod-in-page-banner">
-    <div class="soft-mod-cheat-sheet-cta-block-image"></div>
+    <div class="soft-mod-cheat-sheet-cta-block-image-{{ site.baseurl | remove: '/' }}"></div>
```

Comment:
Does this work in production? I think `site.baseurl` is empty outside of local development

Reply:
If it doesn't work you might be able to use `site.lang` but it would be good to have this code wired into the website someone to be able to check